.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
 rvm:
   - 2.0.0
+  - jruby-9.1.5.0
```

Comment:
I take it you fixed the jruby warnings by using a different jruby version? Cool.